docs/guides/admin/docs/workflowoperationhandlers/cut-marks-to-smil-woh.md

```diff
@@ -0,0 +1,66 @@
+CutMarksToSmilWorkflowOperationHandler
+===================================
+
+Description
+-----------
+
+This operation parses a JSON containing cut marks into a SMIL that can be used by the 
+[VideoEditorWorkflowOperation](editor-woh.md). It does this by attributing the given times to the tracks in the 
+given presentation and presenter flavors. 
+
+## Parameter Table
+
+|configuration keys         |example                |description                                                    |
+|------------------         |-----------            |---------------------------------------------------------------|
+|source-presenter-flavor    |`presenter/prepared`   |The flavor of the presenter video track. Must contain exactly one file.                   |
+|source-presentation-flavor |`presentation/prepared`|The flavor of the presentation video track. Must contain exactly one file.                  |
```

Comment:
I changed it to support a list of flavors (`a/x,b/x`). Does not support wildcards, because I could not figure out how to derive a list of flavors from them. Which I would like to do to check for the amount of tracks in each flavor, because the WHO does not know how it should handle multiple tracks per flavor.